### ISSUE TITLE
Ability to change volume of more sounds

### ISSUE THREAD

**Issue by Archomeda** [Archomeda/csl-ambient-sounds-tuner#14]
Depends on #13.

There are more sounds of which the volume can be changed, but have not yet been added to the mod. The following sounds been proposed to include in the mod:
- Animals
  - [x] Cows
  - [x] Pigs
  - [x] Seagulls
- Buildings
  - [x] Advanced Wind Turbine
  - [x] Airport
  - [x] Bus Depot
  - [x] Cemetery (multiple versions, depending on map theme)
  - [x] Coal/Oil Power Plant
  - [x] Commercial
  - [x] Crematory (multiple versions, depending on map theme)
  - [x] Elementary School (multiple versions, depending on map theme)
  - [x] Fire Station
  - [x] Fusion Power Plant
  - [x] Harbor
  - [x] High School (multiple versions, depending on map theme)
  - [x] Hospital
  - [x] Hydro Power Plant
  - [x] Incineration Plant
  - [x] Industrial
  - [x] Metro Station
  - [x] Nuclear Power Plant
  - [x] Police Station
  - [x] Small Power Plant (?)
  - [x] Solar Power Plant
  - [x] Train Station
  - [x] University (multiple versions, depending on map theme)
  - [x] Water Drain Pipe / Treatment Plant
  - [x] Water Pumping Station
  - [x] Wind Turbine
  - [x] On Fire
  - [x] On Upgrade
- Misc
  - [x] Bulldozer (Buildings)
  - [x] Bulldozer (Props)
  - [x] Bulldozer (Roads)
  - [x] Placement (Buildings)
  - [x] Placement (Props)
  - [x] Placement (Roads)
  - [x] Road Drawer
  - [x] UI Clicks
  - [x] UI Clicks (Disabled)
  - [x] Zone Filler

**Comment by originalfoo** [comment]
Might as well add pigs an cows as well (I've never actually made farmland in the game so don't know if those animals make sounds or not lol).

Trains are another one - I'd actually like to make them a little louder.

Would also be super-cool if there was a way to use custom sound files (.ogg format) similar to V10Siren mod; especially if those sounds could then be shared on workshop as modlets.

**Comment by Archomeda** [comment]
Yeah, those animals make sounds ;) But if I recall correctly, the pigs and cows both fall under the agricultural category. But it's been a couple of weeks since I've touched Cities Skylines, so I need to double check it.

Trains are already listed as Train Movement. They can easily be set louder if needed. I basically took the vanilla volume as maximum, but I can change that.

Custom audio files would be really awesome, but I would like to complete the volumes first before I head that way :smile:.

**Comment by Archomeda** [comment]
Okay. I've added as much sounds as I could. Let me know if you have any more suggestions.

**Comment by originalfoo** [comment]
Menu click sounds, road build sounds, etc? Also bulldoze sound ( may cause incompatibility with https://github.com/V10lator/V10Bulldoze )

**Comment by Archomeda** [comment]
Menu click sounds, hmm... that's a good one, I have not yet found where the global setting is hiding. I'll take a look at that later.

With road build sounds I assume you mean the sounds you hear when placing roads, etc? I have already added these under Misc. Same for bulldozing various objects. Also, V10Bulldoze is just about automatically bulldozing stuff, not changing the sounds, if I'm not mistaken?

**Comment by originalfoo** [comment]
This mod might give some clues to menu sounds: http://steamcommunity.com/sharedfiles/filedetails/?id=431539115

As for v10Bulldose, v10 was going to add option to toggle the bulldoze sound on or off as it was somewhat distracting due to all the auto-bulldozing.

**Comment by Archomeda** [comment]
Ahh. I did not know he was planning to add that. Incompatibility with a mod that overlaps features but also has unique features is not something that I want. I'll see how it goes.

About the clicking sounds, I said I was going to look at it later, but I couldn't resist and added it already. I had to do some detouring in order to make the volume configurable though :disappointed:. But hey, it works.

Now I have to clean some stuff up and fix some stuff as well, since it would seem I get some exceptions.
